app/assets/images/payment_icons/ebucks.svg

```diff
@@ -0,0 +1 @@
Hus8aJ QkjUhK/FddmE/+/ENebsdn5CyOEYTgyxi5y8LrFY2Oi95/Rzm8E5tkL6ZUWkQnpPz+FtcEVDJiFk 9zYYguCEXtGLdJPBMSqGiYEGx2UcHBjnGjwur6kJD5UeMDKdD2eFYbi1oaAnMuM2p8lFvTTc2LhY DEyzNjQ9ZyGkl+KNEb4G0Uh96XpNqA/iJgohF21zFjsG5kmvNVA3B+GWWgWVKEQIuUg9EL4LZdyE st5mjm7GoGoIMdc+Im5w7Mn6WGsWwuGPfA83jvdQ5tnIsxfC5CeE99cdgbJ/JjVj5zfFKQWifjk5 4pNVVsNzbDVEd5ivi6yN9pLNTwqUA5mEkKitexX/fyM1Y/5kjmyGfnC+7hn9hEMMI7kJAQyGM6l6 jVzc+rBg/RMQw/fgBbwPI59tjnrQYhuG91l3pjnigUtztKHlKIR8+oKPbVEEYuFjoIhwPWG8EHIx cs3YhVrMbotbfFz3yPclgDKyi8xusw2FCCFRE74E57saHuHzTTOuMVRzSUl4nieU4/fTx1qzEDLt YD0Xk3Ocj136NGOE4bd0W0tGsO4qXHOuAX3SHCkN9KsC/XI1PMPVNgN0bB3EzWpmNMvL0UuCbELI cR8+GM/uy0iIGpd0cJeRGgidNkB04/hoWAzD0DWl2Oj1fWD8rG5QP32BLnGoAcZytTnqgf+xi8Uu LSdX9HO89Rfj+4v4zoXKiUKohc4ihIS3PIXd2v8hziXwMo/Scbm+kF2weCFkt57eqF48XX8bwh0p w8Ydi/MeqoWcS37il/PEIxch5AYEyULIrn9tPZcsbcA5hLTnxwXU3pABF0xzTPZmE3qzEFbXPdui QsglTMlCSOg1pPUv4bMBeaZuukAhrKn7Cmn0xXnhOrMd1A9AWO5Cw/WeC5u831ICBsbdrIeD5X3H iWNOxE3qdVUhh5vLVV7QI6HY1NR9qp95TQYbPhswhVALlR7YXonwXFYzPmV9IZ/sqAnVIxzX+L2W sCyiOnwMDI/xl6Qse2E4ipO3lRWfRIHQcU1i6FYYzQzvd/gUHbZ//23x/TPwZZQ9dbduz/MbhP+/ RLyNSHMhfi+AJzoBRn0djq/Fseqmbj1nS4P1z8DQ+ZQH4yAvGD3X6Nm61zF4zxpzUfhIvdQoHoGx XIfHfO4zRxJRGzoO5/Uo4nIMdqWXJwUQ3fFA/U1a5GKoCf9bpxUYOwmCtbk+9fKm8FhThsR9KoeN OxjpzcV1eiqtEHL4Qt8A4ZkmLxqPB7vrTCdQNz/hqaEYKKLBUBWuHxfkr0H5hzUJKwWZIuiJ+2Zy DShXI3i9ivP0Takc0GJYJU/YDNGx5cgZ4k1+Sd3Jo3zYAsZ+pNTAC0reMCEGvdwBgjMC3dZaeGl8 npaLlW0zqlzcfB83Zqg/UQ+Ex4PpaDGE8Ni8LG7OQHGprrsI5Tlfjx/pJ07gZXLZzeaNG7bQj77R 00s3wE4PrZoTNGMvkBF8frn+JO9ZaqQR20A1thCbQqG9njF/0XkHx3bTQpVpxpjgchCmxYXYyWJD L5n/cfOIdGB5ho75nVefrNf6M/UTK8mLs8dM2NVLCzcPPtUSA/McMfIQ/R9vQPHgk0E1SDv5hhMP jo1yDWDyzSwZNW9sJUNDh+v00okqbz5MK1a3MS+YT6LweO24PyaQ15mLx7kAPp3HXSpwcwZ4hVPc Rq4tT3iBfHJkLh+LNJfHwcGhXIj2lh/DACe7NYYtS9T/F1FutFrOGWIHBwcPfIA/6pdfaM/Q7V/Y Ioz4ZD5E8Ewngg4OLQiKoeot+zVWyXM2Q3UsHSOV8jG7wyV/htjBwSE3qC6yLSdQXDe59KT3HfHL nAafeI+jOTg4tB6suVi+DyEcAUN16wxLRE5OoX5fb/DLCabaHRwcWhuivWWPiE+uh7GusxmyY+GM VMkm1Ov4qE9+ZqrbwcGhtUJ1lq2jFXJapFK+tBm0Y/6EAK4FB0cvlea/t9XBwaF8iFbJURDD/7px w8Kpu8I++Qbe4BXRXpL6fKWDg0PrR7RSDoIh34Pu8jc2Q3dMT3iAq8HHo73lJFOdDg4ObRXwDHeA IJ4W8ctroHsSJQdGKmQp6u0G1FvqFvEODg5tE2bx9d7wcIa5rnJmRirlddTVr1Vnye19Fw4ODm0L EMQtYeSnqwr5L7rMqyCM7okUEDeHBtTF+6ib60r2AnYHB4fWBb5HI+oTf8Qnb3BCwCYOHYV8TK6x SoZqL7C/e1TOwaFDgctsNlTKQRAAn35czCIS7Zp+WYGbQA3O/+hoX0ncjNPBwaHjIXqh7IKu4UBw kV4y0g67zDwnMsJusF+ejF7RonsHOjg4tFaoCvlBpI9cC8F4GVzSXrrNEL/1kSr5QHuAPvmtOV0H BweH9IhWyQ8jPjkXYlgNvt/YBl8NgDI3osu/EOL3aKRCfBwDBEvzmkEHB4f2Cb3kpq/sBPH4Ebyo iyAs3N1miU10WhMhfuvgAb4OAR/A8T9VKbu5pTAODg5FAzd1gLd4IUTmKYjNAnyuhTg2QHyijVXl HVdE3hzziyD/9ch7JcowC2I9MHqFpH+3g4ODg0OxQA8r2ksOhMf1bwjR7eBDEKUXOfsMUVqM3xsp VDYBK5Ra9Cq14H2Oz1nIY1JjhQxDF/5SeKy/ip7hur0ODg4tBL1L9uWyI8cVo5VybEOl/BPeog9C dStEsR6i9Ry9NQjWR3rNXqUsxO+lRtRWa/plFT6Xg4vALxH3E/AdhH8Bn48inXshdtdEest5DT45 Ed9/pi6T77ot8h0cHFotjMe4Ddgp2lN2hUh+D+L1fXiQe0f7yT7rest+fMUAju9PrgP170rZd20v 2Qdx9mI3XI/vVch3OMmheshWFF2ThYODQ5uAyP8Ddgv2nGeGL3wAAAAASUVORK5CYII=" height="20" width="34" id="pi-ebucks-image193" x="1.729" y="2.034"/></svg>
```

Comment:
Please remove all `id` attributes except the `<title>` element as there are not necessary.

[![alt](https://screenshot.click/07-39-kwu8y-42jsv.png)](https://screenshot.click/07-39-kwu8y-42jsv.png)
[![alt](https://screenshot.click/07-41-ix76i-n7k2w.png)](https://screenshot.click/07-41-ix76i-n7k2w.png)

Reply:
I have actioned your comments. Thanks for the review.